CHANGES.rst

```diff
@@ -22,6 +22,9 @@ resample
   to ResampleStep, and another where photometry keywords weren't being updated
   correctly to reflect the correct pixel scale ratio. [#7033]
 
+- Changed computation of meta.resample.pixel_scale_ratio to correctly be the
+  ratio of input / output areas [#7048]
```

Comment:
I think you could have simply appended 7048 to the line above: `... pixel scale ratio [#7033, #7048]`